pandas/core/frame.py

```diff
@@ -1467,10 +1467,10 @@ def to_csv(self, path_or_buf=None, sep=",", na_rep='', float_format=None,
         encoding : string, optional
             A string representing the encoding to use in the output file,
             defaults to 'ascii' on Python 2 and 'utf-8' on Python 3.
-        compression : string, optional
-            a string representing the compression to use in the output file,
-            allowed values are 'gzip', 'bz2', 'xz',
-            only used when the first argument is a filename
+        compression : {'infer', 'gzip', 'bz2', 'xz', None}, default None
```

Comment:
'zip' is missing here? 
(unless it is wrongly included in the docstring of get_handle)

Reply:
I already removed 'zip' from the docstring.

Reply:
What do you mean exactly? Where did you remove it?

I don't know whether `zip` is working for to_csv, therefore asking it. I just see that the line below mentions 'zip', but the line above not. And `get_handle` docstring also mentions 'zip'

Reply:
Quickly tested, and 'zip' is not supported for writing, only reading. So I think it should be removed from the line below.

Reply:
I meant my docstrings for to_csv and to_json do not include zip anymore. And like you said, get_handle only supports zip for writing. So maybe only the docstring for get_handle should clarify that.